rest_framework_swagger/views.py

```
@@ -41,7 +44,12 @@ def get_restructuredtext(view_cls, html=False):
 
 class SwaggerUIView(View):
 
-    def get(self, request, *args, **kwargs):
+    def get(self, request, api_version=None, *args, **kwargs):
+
+        if api_version not in SWAGGER_SETTINGS.get('available_api_versions') and SWAGGER_SETTINGS['api_version']:
+            return redirect(reverse('django.swagger.base.view', args=(SWAGGER_SETTINGS['api_version'],)))
```

This should maybe raise an error instead?